web/src/app/modules/shared/components/presentation/overflow-labels/overflow-labels.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="labels-wrapper" *ngIf="labels">
+  <ng-container *ngFor="let label of showLabels; trackBy: trackByIdentity">
+    <span
+      *ngFor="let item of label | keyvalue; trackBy: trackByIdentity"
+      class="label label-light-blue clickable"
```

Comment:
I would recommend extracting out the the duplicating code segment that shows group of labels - notice that lines 2-9 are same as 18-25.

Reply:
I guess we could extract those lines into a child component and reuse it, but I think that would require more work and even more lines of code as it is right now.